internal/cmd/operator-sdk/cli/cli.go

```diff
@@ -83,9 +83,33 @@ func GetPluginsCLIAndRoot() (cli.CLI, *cobra.Command) {
 	}
 	root.PersistentPreRun = rootPersistentPreRun
 
+	// Hidden commands which are based operator type
+	hiddenCommands(root)
+
 	return c, root
 }
 
+// hiddenCommands will hidden commands based on the operator type
+func hiddenCommands(root *cobra.Command) {
```

Comment:
Hiding commands breaks the spirit of plugins. After running `init` with a non-Go plugin, `edit` won't be available. The default plugin is Go, so `edit` should always show up if you run `kubebuilder -h`; hiding the command because other plugins don't implement it doesn't fit into the plugin model.

Reply:
Since shows that @joelanford is ok with it. we can just remove this one.

Reply:
I think we need to discuss that since it is very strange in the user point of view we run `operator-sdk -h` then it shows a command e.g edit and then when we try to use it check the msg that the command does not exist. However, it is out pf the scope. I will raise an issue for that.